org.serioussam.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,118 @@
+      - install -Dm 755 SamTSE/SE1_10b.gro /app/share/serioussamse/SE1_10b.gro
+    sources:
+      - type: archive
+        url: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/1.10.6d.tar.gz
+        sha256: 768ea6a42e6506ab4619771932032a392c6e3e64be9ecc694b02a172b767b2fa
+        x-checker-data:
+          type: anitya
+          project-id: 9154
+          url-template: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/$version.tar.gz
+      - type: shell
+        commands:
+      # ----------------------------------------------------
+      # This commands adapted source code for create flatpak
+      # ----------------------------------------------------
+      # For amd64/i686 architecture (for any computers)
+          - find . -name "CMakeLists.txt" -exec sed -i 's/-march=native/-mtune=generic/g' {} +
+      # Remove ImageMagick depend (no need this depend for flatpak)
+          - find . -name "CMakeLists.txt" -exec sed -i 's/if(NOT LOCAL_INSTALL)/if(LOCAL_INSTALL)/g' {} +
+      # Replace /usr with /app
+          - find . -name "Engine.cpp" -exec sed -i 's/\/usr\//\/app\//g' {} +
+      # Fix /app/lib (replace "//" with "/")
+          - find . -name "Engine.cpp" -exec sed -i 's/\/app\/lib\/\//\/app\/lib\//g' {} +
+      # sys_iSysPath == 0 (force library path)
+          - find . -name "Engine.cpp" -exec sed -i 's/sys_iSysPath == 0/sys_iSysPath == 1/g' {} +
+      # _fnmModLibPath = _fnmApplicationPath (force library path)
+          - find . -name "Engine.cpp" -exec sed -i 's/_fnmModLibPath = _fnmApplicationPath/_fnmModLibPath = "\/app\/lib\/" + strGameID + "\/"/g' {} +
+      # Fix SDL2 message box
+          - find . -name "Engine.cpp" -exec sed -i 's/~\/.local\/share\/Serious-Engine\///g' {} +
+          - find . -name "Engine.cpp" -exec sed -i 's/_fnmUserDir,(const char \*) strGameID/_fnmUserDir,(const char \*) _fnmUserDir/g' {} +
+      # supresses warnings for a non-used directory
+          - mkdir -p /app/share/serioussam && mkdir -p /app/share/serioussamse
+          - mkdir -p /app/lib/serioussam && mkdir -p /app/lib/serioussamse
+
+  - name: launcher
+    buildsystem: simple
+    sources:
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.SamTFE.desktop
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.SamTSE.desktop
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.metainfo.xml
+      - type: file
+        url: https://raw.githubusercontent.com/tx00100xt/SeriousSamClassic-VK/main/SamTFE/serioussam.png
+        sha256: e12fdccc0ffbbd64ad2d5459d6b1da811fbe099fe8fefc0172bbf7d835648923
+        dest-filename: org.serioussam.SeriousSamClassic-VK.png
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.128.png
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.64.png
+      - type: file
+        path: org.serioussam.SeriousSamClassic-VK.48.png
```

Comment:
you don't need these icon size. they don't come from upstream? They shoudln't be here.